src/styles/shared/_interaction-state.scss

```diff
@@ -24,3 +24,16 @@
   }
   background-image: $backgrounds;
 }
+
+@mixin list-single-selection-state {
+  background: var(--p-interactive);
```

Comment:
I prefer the explicitness of `background-color` if we know that we are dealing with exactly that - a `color` and not some other `background-*` property.